src/main/java/qna/domain/Answer.java

```diff
@@ -78,12 +84,7 @@ public DeleteHistory deleteAnswer(User loginUser) {
         }
 
         setDeleted(true);
-        return DeleteHistory.builder()
-                .contentType(ContentType.ANSWER)
-                .contentId(getId())
-                .deletedBy(getWriter())
-                .createDate(LocalDateTime.now())
-                .build();
+        return DeleteHistory.of(ContentType.ANSWER, getId(), getWriter(), LocalDateTime.now());
```

Comment:
이 부분도 인수를 좀 더 줄여볼 수 있을꺼 같아요! 😄 
```suggestion
                .addHistory(DeleteHistory.ofAnswer(getId(), getWriter()));
```